lib/config/definitions.js

```diff
@@ -1630,6 +1630,21 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'homebrew',
+    releaseStatus: 'unpublished',
+    description: 'Configuration object for homebrew',
+    stage: 'package',
+    type: 'object',
+    default: {
+      enabled: false,
+      commitMessageTopic: 'Homebrew Formula {{depName}}',
+      managerBranchPrefix: 'homebrew-',
+      fileMatch: ['^Formula/[^/]+[.]rb$'],
```

Comment:
Is it a restriction that formulas must live within `/Formula/*` within a repo, or might there be some nested, e.g. under `/foo/Formula/*`?

Reply:
I couldn't find anything about that in the documentation, so I don't know if `homebrew` would work if `Formula` directory is not at the root. Most custom taps I looked at had `Formula` directory at the root, see: https://docs.brew.sh/Interesting-Taps-and-Forks#interesting-taps--forks.

Although I have found some taps without the `Formula` directory, with just `*.rb` files at the root:
https://github.com/titanous/homebrew-gnuradio
https://github.com/sidaf/homebrew-pentest
but I think it would be unreasonable to `fileMatch` all `*.rb` files at root, and it would be easy to move all those `*.rb` files to a `Formula` directory.